src/components/ExpandableGrid.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { DataView } from '../types';
+import { ExpandableGridRow } from './ExpandableGridRow';
+
+interface ExpandableGridPops {
+    dataView: DataView;
+    network: string;
+    data: Object[];
+}
+
+interface ExpandableGridState {
+    expansions: Map<string, string>;
+}
+
+export class ExpandableGrid extends React.Component<ExpandableGridPops, ExpandableGridState> {
+
+    constructor(props: ExpandableGridPops) {
+        super(props);
+        this.state = {expansions: new Map<string, string>()};
+    }
+
+    public render() {
+        if (this.props.data.length !== 0) {
+            return (
+                <table>
```

Comment:
I'm not a huge fan of tables - I prefer css display grid and flexbox. I find that makes responsiveness easier, but personal preference. Also, if this table gets very big, you might need to consider pagination or virtualization